### PR TITLE
Add extra pages

### PR DESCRIPTION
**Pull request by sergio-costas**
These pages are required for ubuntu core desktop

### CODE REVIEW

**Comment by sergio-costas** [comment]
@kenvandine When you have some spare time, please review this.

**Review by kenvandine — APPROVED** [review]
Looks good, but we can't merge this until launchsession lands in ubuntu-desktop-provision

**Review by kenvandine — APPROVED** [review]
Thanks, this is good until we can sort out the new page in ubuntu-desktop-provision

**Pull request merged**
